pinot-server/src/main/java/com/linkedin/pinot/server/starter/helix/HelixInstanceDataManager.java

```diff
@@ -231,17 +231,11 @@ public synchronized void removeSegment(String segmentName) {
   }
 
   @Override
-  public synchronized void reloadSegment(@Nonnull SegmentMetadata segmentMetadata,
-      @Nonnull CommonConstants.Helix.TableType tableType, @Nullable TableConfig tableConfig, @Nullable Schema schema)
+  public synchronized void reloadSegment(@Nonnull String tableNameWithType, @Nonnull SegmentMetadata segmentMetadata,
+      @Nullable TableConfig tableConfig, @Nullable Schema schema)
       throws Exception {
     String segmentName = segmentMetadata.getName();
-    String tableName = segmentMetadata.getTableName();
-    if (tableType == CommonConstants.Helix.TableType.OFFLINE) {
-      tableName = TableNameBuilder.OFFLINE.tableNameWithType(tableName);
-    } else {
-      tableName = TableNameBuilder.REALTIME.tableNameWithType(tableName);
-    }
-    LOGGER.info("Reloading segment: {} in table: {}", segmentName, tableName);
+    LOGGER.info("Reloading segment: {} in table: {}", segmentName, tableNameWithType);
```

Comment:
please also add an info log when reload is successful with segment name and table name in the log.